docs/topics/auth/default.txt

```diff
@@ -580,7 +580,7 @@ The permission_required decorator
     (i.e. ``polls.can_vote`` for a permission on a model in the ``polls``
     application).
 
-    The decorator may also take a list of permissions.
+    The decorator may also take an iterable of permissions.
```

Comment:
Need a `.. versionchanged:: 1.9` annotation describing the change so readers of the dev docs with an older Django version don't get confused.